src/components/TabulatorComponent.vue

```diff
@@ -23,6 +24,58 @@ export default class TabulatorComponent extends Vue {
 
   private tabulatorInstance: Tabulator | null = null;
 
+  get eventOptions(): object { 
```

Comment:
Maybe we can split that code in another file, to manager only the row events

Reply:
I agree. The file was getting rather long with it added this way. Was just happy to have it working. Will look to refactor.

Reply:
Here is the new factory function that creates events simply based off of the option name. Provide an array of option names and we can create all events this way. 

```ts
export default function (this: TabulatorComponent, events: string[]): Object {
  const vm = this
  let obj: {[index:string] : {}} = {}
  events.forEach(event => {
    obj[event] = function(...args: any[]) {
      vm.$emit(dashify(event),...args) 
    }
  })
  return obj
}
```